.github/workflows/renovate.yaml

```diff
@@ -46,5 +42,5 @@ jobs:
         env:
           RENOVATE_REPOSITORIES: ${{ github.repository }}
           RENOVATE_ONBOARDING: "false"
-          RENOVATE_GIT_AUTHOR: "${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"
+          RENOVATE_GIT_AUTHOR: "${{ github.actor }} <${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com>"
```

Comment:
this field needs a [RFC5322](https://datatracker.ietf.org/doc/html/rfc5322)-compliant string https://docs.renovatebot.com/configuration-options/#gitauthor